### PR TITLE
Improve SEO more

### PR DESCRIPTION
**Pull request by RDIL**
edit: ready for merge 😄

### CODE REVIEW

**Comment by fkorotkov** [comment]
Is it ready for review? I see you are making changes...

**Comment by RDIL** [comment]
Nope, much more coming, I'll let you know when it is, need to keep it open for the patch branch

**Comment by fkorotkov** [comment]
Got it. Please use [drafts](https://github.blog/2019-02-14-introducing-draft-pull-requests/) next time. 👍

**Comment by RDIL** [comment]
Okay 👍

**Comment by RDIL** [comment]
@fkorotkov done!

**Comment by fkorotkov** [comment]
Great! Thanks! Will merge it tomorrow along with #37

**Comment by RDIL** [comment]
Will do! :D

**Comment by RDIL** [comment]
@fkorotkov looks like this can now be merged.

File: public/robots.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: *
+Disallow: 
+Sitemap: https://cirrus-ci.com/sitemap.xml
```
Comment (RDIL) [inline public/robots.txt]: This should allow search engines to crawl all pages making the search ranking higher

Comment (fkorotkov) [reply public/robots.txt]: There is no https://cirrus-ci.com/sitemap.xml and I also think engines are crawling it anyway. `robots.txt` is usually used to block some parts from the engines. As far as I know engines are crawling by default.

**Comment by RDIL** [comment]
I can remove the sitemap, but I suggest keeping the robots file because it improves the sites score.

**Comment by fkorotkov** [comment]
Okay. Let's keep it. 🙌

**Pull request merged**